src/_utils/item/Item.tsx

```diff
@@ -101,6 +101,24 @@ export const Item = (props: ItemProps) => {
 
   const getTextColor = (textColor: string) => (disabled ? color.gray : textColor)
 
+  const getLeftTitle = (value: any) => {
+    if (typeof value === 'object') {
+      return leftTitle
```

Comment:
I find this way too flexible, I think the component API should be more restrictive.
I'd remove this, and use `leftTitleDisplay` prop instead, wdyt? So we can keep `leftTitle` prop as string.

Reply:
I could make it restrictive to any typography component. 
But you are right that **ONLY** `BigData` uses `Display2` and all other `Item` use `Title`.
I just don't like the idea to have one prop by component...

Reply:
> I just don't like the idea to have one prop by component...
What do you mean?

Reply:
Creating `leftTitleDisplay` for instance because the API goes with implementation details instead of what is the function of this property which is to have a `LeftTitle`. 

If you want to keep `Title` and do not touch `Item` API. I can override on `BigData` CSS.  It will have less impact, I guess.

_(really sad though, I can't override the component itself since this component doesn't follow a `styled-components` syntaxe)_

**Should I override title by CSS**?

Reply:
I would go for a list of typogaphic styles as a param type (VS CSS or having separate pair of props for each styles).

Reply:
I think ultimately we need to get rid of these leftTitleDisplay props: the combinatory explodes and allow to create typographic styles that don't exists (since you can change independantly font size/color/etc)

Reply:
[Compound components](https://www.google.fr/search?q=react+compound+components) would resolve this kind of issues 😉